### PR TITLE
MAIN - Add Sidebar Component

### PR DESCRIPTION
**Pull request by Weetbix**
Adds a Sidebar component which allows the user to:
- Customize header (logo, content, link)
- Add links with icons

@lanwen would you mind taking a look at this when you get time? (non-urgent)

### CODE REVIEW

**Comment by paschalidi** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Big PR - <i>This PR is unlikely to get reviewed because it touches too many lines 
  (3049). 
  Consider sending smaller Pull Requests and stack them on top of each other.</i></td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>✅ Get a 👍 for requesting reviewers. - <i>Celebrating every win!</i></td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>✅  Get a 🏆 for adding description on this PR. - <i>Celebrate on every step!</i></td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against b0e58edd52b765707d2e42d27e5cba6661e9e483
</p>

File: src/Sidebar/sidebar.style.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import styled, { css } from "styled-components";
+import colors, { hexToRGBA } from "../Colors";
+
+export const SidebarStyles = styled.div`
+  background-color: ${colors.textPrimary};
+  line-height: 1.38;
+  color: #fff;
```
Comment (lanwen) [inline src/Sidebar/sidebar.style.js]: shouldn't it come from the color constants?

**Comment by lanwen** [comment]
Screenshot please?

**Comment by Weetbix** [comment]
@lanwen screenshot is in the PR at https://github.com/UvitaTeam/vivy-components/blob/b0e58edd52b765707d2e42d27e5cba6661e9e483/tests/__image_snapshots__/storyshots-puppeteer-test-js-image-storyshots-sidebar-basic-1-snap.png

🙂

**Pull request merged**